service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/prepopulator/modifier/ManageOrdersCaseDataFixer.java

```diff
@@ -48,4 +49,11 @@ public CaseData fix(CaseData caseData) {
 
         return caseData;
     }
+
+    public CaseDetails fixAndRetriveCaseDetails(CaseDetails caseDetails) {
+        if (!AMEND.equals(caseDetails.getData().get("manageOrdersOperation"))) {
```

Comment:
I don't think this will ever be true until you actually serialise the field, otherwise it will just be a string and AMEND is an enum

Reply:
resolved this one as discussed